lib/server.js

```diff
@@ -169,22 +169,31 @@ server.onPhantomPageCreate = function(req, res) {
             resourcesRequested: [],
             resourcesReceived: [],
             resourcesTimeout: [],
+            resourcesData: {},
             lastResourceReceived: null
         };
 
         this.onResourceRequested = function(requestData, request) {
+            _this.prerender.resourcesData[requestData.url] = {
+                start: null,
+                end: null,
+                blocked: false,
+                error: false,
+                timedOut: false
```

Comment:
maybe it would be possible to mutualize blocked, error and timeOut in a numerical value ? status code's like ? or an enum 'status' with 'success' 'blocked', 'error', 'timeout' ?